Cesium.externs.js

```diff
@@ -1499,6 +1507,23 @@ Cesium.optionsPolylineGeometry;
 Cesium.PolylineGeometry = function(object) {};
 
 
+/**
+ * @typedef {{
+ * rectangle: {Cesium.Rectangle},
+ * vertexFormat: number
+ * }}
+ */
+Cesium.optionsRectangleGeometry;
+
+
+/**
+ * @constructor
+ * @param {Object=} object
```

Comment:
Hmm, I see you actually created the typedef but did not used it. It should be used instead of `object`.
I checked the code of the extern and there are other places like this; these are very weird errors and need a fix.